### PR TITLE
Disable input sounds when in password mode

### PR DESCRIPTION
**Pull request by Animus-Surge**
#1039 

When password mode is enabled, I've disabled the sounds that get played when pressing a key. This included the stdout sound that gets played when the terminal recieves another piece of text as I found that when typing in the terminal, it played the sound. I will make changes if requested/as needed.

small edit: I'm not sure why the commit log shows stuff from my last PR, but there shouldn't be any changes made to that part eDEX

### CODE REVIEW

File: src/classes/keyboard.class.js [hunk 1]
```diff
@@ -343,9 +344,8 @@ class Keyboard {
                 }, 100);
             }
 
-            if (e.key === "Enter") {
+            if(this.container.dataset.passwordMode == "false")
```
Comment (GitSquared) [inline src/classes/keyboard.class.js:347]: Shouldn't this be
```suggestion
            if(this.container.dataset.passwordMode == "false" && e.key === "Enter")
```
?

Comment (Animus-Surge) [reply src/classes/keyboard.class.js:347]: Yes, my bad

**Comment by GitSquared** [comment]
Thanks for working on this!

**Comment by Animus-Surge** [comment]
No problem!

**Pull request merged**